site/blog/starlingx-meetup-september-2019.md

```diff
@@ -0,0 +1,48 @@
+
+**"MEC (Multi-access Edge Computing) platform and implementation by China Unicom Research Institute":**
+The definition of MEC environments has been driven by standards bodies such as 3GPP and ETSI. Following the guidelines of the ETSI MEC architectural framework, China Unicom proposed their reference design on several layers: OSS/BSS, MEAO(MEC Application Orchestrator)/NFVO, VNFM, and MEPM(Multi-access Edge Platform Manager)/MEP(Multi-access Edge Platform). China Unicom chose StarlingX as infrastructure solution for the virtualization layer for its Edge Business Platform to fulfill the requirements on high bandwidth, low latency and massive scale of connections. The presenter also mentioned the importance of StarlingX being fully aligned with ETSI NFV guidelines besides the characteristics that were designed with full dedication to edge use cases. 
+
+**"Edge computing research to empower financial applications by China UnionPay and Topic":**
+By nature, financial applications (for instance, payment use cases) have special requirements, such as increased security and privacy, dynamic and fragmented workloads, and so forth. During this session, the audience could learn about China UnionPay's official designated vendor to Chinese banks) vision and technical approaches how they would utilize edge computing platform to empower typical financial applications. StarlingX was used in a proof of concept deployment of a contactless payment system for electronic-vehicle charging stations as the secure and scalable edge platform to host video surveillance workloads, charge metering and secure payment modules. In the context of deploying financial applications on the edge, StarlingX is an outstanding solution benefiting from its hardened security design (specifically with the well integrated TPM framework) that is backed by security technologies like Intel SGX on Intel platforms.
+
+You can learn more about the contactless payment system use case and deployment work at a [presentation](https://www.openstack.org/summit/shanghai-2019/summit-schedule/events/23945/secured-edge-infrastructure-for-contactless-payment-system) at the Open Infratsructure Summit in Shanghai.
+
+**"Heterogeneous and Collaborative Management Requirements on Cloud and Edge China Mobile Research Institute":**
+Such as other carriers, China Mobile is also aligning their edge platform with the ESTI MEC framework and reference architecture, though there is a certain level of difference in implementation details. Facing the natural complexity of edge clouds, in terms of multiple locations (cross regions), heterogeneousness, collaboration, and autonomy, the audience could learn about China Mobile's experience about working with these challanges that they learned from their practice. StarlingX was presented on the the infrastructure layer covering the NFVI and VIM components with integrating services from open source projects such as OpenStack and Kubernetes to fulfill edge requrements.
```

Comment:
Such as other carriers, China Mobile is also aligning their edge platform with the ESTI MEC framework and reference architecture, though there is a certain level of difference in implementation details. Facing the natural complexity of edge clouds, in terms of multiple locations (cross regions), heterogeneousness, collaboration, and autonomy, the audience could learn about China Mobile's experience about working with these challenges that they learned from their practice. StarlingX was presented on the the infrastructure layer covering the NFVI and VIM components with integrating services from open source projects such as OpenStack and Kubernetes to fulfill edge requrements.